analyzer/consensus/genesis.go

```diff
@@ -296,7 +295,7 @@ func (mg *GenesisProcessor) addGovernanceBackendMigrations(batch *storage.QueryB
 				proposal.State.String(),
 				proposal.Deposit.ToBigInt(),
 				proposal.Content.ChangeParameters.Module,
-				hex.EncodeToString(proposal.Content.ChangeParameters.Changes),
+				proposal.Content.ChangeParameters.Changes,
```

Comment:
Wdyt of converting the cbor to json here in the analyzer and then storing it as a `JSONB` column in the database? It would be nice to do as much of the processing in the analyzers as possible to keep the api server fast/simple.

Reply:
Hm i think I would slightly prefer having the raw field in the DB, since that matches what was actually proposed. But maybe we could store both the raw cbor and the parsed JSONB.

Maybe we can keep this as a potential followup in future, the number of proposals is really low (4 on mainnet and 11 on testnet) so I don't think this will be problematic anytime soon.
(and even out of those 15 proposals, in total 3 are of the change-parameter types, for which this parsing is relevant).

Reply:
ah makes sense, keeping it as a follow-up sgtm